compute/cker/include/cker/operation/Range.h

```diff
@@ -53,8 +54,6 @@ inline void Range(const T *start_data, const T *limit_data, const T *delta_data,
   {
     output_data[i] = value;
     value += delta_value;
-
-    assert(value < limit_value);
```

Comment:
Can I ask why you removed this? Wouldn't this be double-check `GetSize` result?

Reply:
If `delta_value` is a negative value, this assertion makes an error.

Reply:
also 
considering [start = 1, delta 1, limit = 2]
after 1st iteration output[0] = 1, value == 2, so assert fails.

Reply:
Oh then this assertion should have been at the top of the loop. And should check if `delta_value` is negative or positive.

Reply:
@wateret 
There is this line.
```c++
//compute/cker/include/cker/operation/Range.h
//line 36-39
if (!((start > limit && delta < 0) || (start < limit && delta > 0)))
  {	  {
    throw std::runtime_error("Range: invalid input values");
  }	  }
```

Reply:
The assertion is optional and a defensive code. If there is no change, it will be filtered by the code you said. So you don't have to add it if you think it is duplicated.